pkg/model/constants.go

```diff
@@ -55,3 +55,5 @@ const (
 	ServiceMonitorName                    = ApplicationName + "-service-monitor"
 	MigrateBackupName                     = "migrate-backup"
 )
+
+var PodLabels = map[string]string{}
```

Comment:
I just noticed that this is actually not marked as a constant (as the constants above). Is there any reason for it?

Reply:
A Hashmap can not be a constant.